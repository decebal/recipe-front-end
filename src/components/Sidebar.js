// @flow
import React from 'react';
import Navbar from '@bootstrap-styled/v4/lib/Navbar';
import {
  Container, Nav, NavItem, NavLink,
} from '@bootstrap-styled/v4/dist/@bootstrap-styled/v4';
import OldSchoolMenuLink from './OldSchoolMenuLink';

const Sidebar = () => (
  <Navbar color="faded" light toggleable="lg">
    <Container>
      <Nav navbar className="mr-auto">
        <NavItem>
          <NavLink><OldSchoolMenuLink activeOnlyWhenExact to="/" label="Home" /></NavLink>
        </NavItem>
      </Nav>
    </Container>
  </Navbar>
);

export default Sidebar;
