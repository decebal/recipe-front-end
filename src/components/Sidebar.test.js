import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Sidebar from './Sidebar';


test('Sidebar renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter initialEntries={['/']}>
      <Sidebar />
    </MemoryRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
