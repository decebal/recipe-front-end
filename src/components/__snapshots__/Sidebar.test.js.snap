// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar renders correctly 1`] = `
<nav
  className="Navbar-sc-1o0l5dr-0 LHdqb navbar navbar-toggleable-lg navbar-light bg-faded"
  role="navigation"
>
  <div
    className="Container-u38a83-0 jDuhNh container"
  >
    <ul
      className="mr-auto Nav-ggw7qv-0 bQFVaC navbar-nav Ul-sctud2-0 ldzbsT"
      role="navigation"
      tag="nav"
      toggleable="false"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link A-sc-70cat-0 gsCFrI"
          onClick={[Function]}
        >
          <div
            className="active"
          >
            &gt; 
            <a
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </div>
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
